### PR TITLE
Digital input

### LINKED ISSUE #39: Add visual component for TcoDigitalSensor

### PR DESCRIPTION
**Pull request by jozefchmelar**
Rather simple UI for DigitalInput component 

image

### CODE REVIEW

File: src/TcoCore/src/TcoCore.Wpf/Converters/SignalToBrushConverter.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    public class SignalToBrushConverter : BaseConverter
+    {
+        public override object ToConvert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            try
+            {
+                var signal = (bool)value;
+                if (signal)
+                {
+                    return Application.Current.TryFindResource("SignalOnBrush") ?? Brushes.GreenYellow;
```
Comment (jozefchmelar) [inline src/TcoCore/src/TcoCore.Wpf/Converters/SignalToBrushConverter.cs]: I'd like to discuss usage of a resource here... It's good idea to have same color for digital input in ie. a Cylinder, and here... an app that uses wpf, should define preferably in app.xaml a resource with this name.

Comment (PTKu) [reply src/TcoCore/src/TcoCore.Wpf/Converters/SignalToBrushConverter.cs]: I'd go with resources from the material-design library's colors (Primary/Secondary/Error) and maybe we can add some other resources ('Alert/OnAlert (to capture attention), Active/OnActive (to indicate activity). 
Not a great eye for aesthetics... so open to other ideas :)

Comment (jozefchmelar) [reply src/TcoCore/src/TcoCore.Wpf/Converters/SignalToBrushConverter.cs]: Ok so I managed to find possible brushes in material toolkit that we could use...we'll see along the way.

File: src/TcoCore/src/TcoCore.Wpf/Converters/SignalToBrushConverter.cs [hunk 2]
```diff
@@ -15,11 +22,11 @@ public override object ToConvert(object value, Type targetType, object parameter
                 var signal = (bool)value;
                 if (signal)
                 {
-                    return Application.Current.TryFindResource("SignalOnBrush") ?? Brushes.GreenYellow;
+                    return Application.Current.TryFindResource("SecondaryHueMidBrush") ?? Brushes.GreenYellow;
```
Comment (jozefchmelar) [inline src/TcoCore/src/TcoCore.Wpf/Converters/SignalToBrushConverter.cs:25]: This means Accent color @PTKu

**Comment by PTKu** [comment]
merged in https://github.com/TcOpenGroup/TcOpen/pull/53/commits/b002eb435c96bf618221f9e10e3b230daa62039d
see also #53 
closing here

**Pull request closed without merging**